modules/understanding-upgrade-channels.adoc

```diff
@@ -155,6 +196,19 @@ and participating in Red Hat’s connected customer program. Red
 Hat uses this program to observe the impact of updates on your specific hardware and software
 configurations. Future releases might improve or alter the pace at which updates move
 from the `fast-{product-version}` to the `stable-{product-version}` channel.
+endif::openshift-origin[]
+
+ifdef::openshift-origin[]
```

Comment:
Lets remove the whole section - we don't have a nice way to block the edges right now